frontend/src/scenes/groups/Groups.tsx

```diff
@@ -31,9 +37,20 @@ export function Groups({ groupTypeIndex }: { groupTypeIndex: GroupTypeIndex }):
         )
     }
 
+    let columns = {
+        group_name: {
+            title: groupTypeName,
+        },
+    } as QueryContext['columns']
+    let hiddenColumns = [] as string[]
```

Comment:
```suggestion
    let columns: QueryContext['columns'] = {
        group_name: {
            title: groupTypeName,
        },
    }
    let hiddenColumns = string[]
```

Do you need the casting here? It seems the definitions should be enough